patches/build-util-android_chrome_version.py.patch

```diff
@@ -1,16 +1,36 @@
 diff --git a/build/util/android_chrome_version.py b/build/util/android_chrome_version.py
-index f00cd99d2bfa5793d6ed5d96025885ad83af412f..4e49484e6bdb3648739521c6125fbacb8516c33c 100755
+index f00cd99d2bfa5793d6ed5d96025885ad83af412f..41f44538d04232f990d0af59682b92f43fe1fa23 100755
 --- a/build/util/android_chrome_version.py
 +++ b/build/util/android_chrome_version.py
-@@ -199,6 +199,7 @@ def _GetAbisToDigitMask(build_number):
+@@ -48,6 +48,19 @@ For example:
+ import argparse
+ from collections import namedtuple
+ 
++import os.path
```

Comment:
I'd like to find some other way to do this like maybe an android studio config file? Or @goodov maybe we should change our python path and other overrides into a file you can source from the shell?

Reply:
maybe have it generate an env/env.bat file at runtime